cdp-agentkit-core/python/cdp_agentkit_core/actions/superfluid/create_flow.py

```diff
@@ -0,0 +1,81 @@
+from collections.abc import Callable
+
+from cdp import Wallet
+from pydantic import BaseModel, Field
+
+from cdp_agentkit_core.actions import CdpAction
+from cdp_agentkit_core.actions.superfluid.constants import (
+    CREATE_ABI,
+)
+
+CREATE_FLOW_PROMPT = """
+This tool will create a money flow to a specified token recipient using Superfluid. Do not use this tool for any other purpose, or trading other assets.
+
+Inputs:
+- Wallet address to send the tokens to
+- Super token contract address
+- The flowrate of flow in wei per second
+
+Important notes:
+- The flowrate cannot have any decimal points, since the unit of measurement is wei per second.
+- Make sure to use the exact amount provided, and if there's any doubt, check by getting more information before continuing with the action.
+- 1 wei = 0.000000000000000001 ETH
+"""
+
+class CreateFlowInput(BaseModel):
+    """Input argument schema for creating a flow."""
+
+    recipient: str = Field(
+        ..., 
+        description="The wallet address of the recipient"
+    )
+
+    token_address: str = Field(
+        ..., 
+        description="The address of the token that will be streamed"
+    )
+
+    flow_rate: str = Field(
+        ..., 
+        description="The flow rate of tokens in wei per second"
+    )
+
+def create_flow(wallet: Wallet, recipient: str, token_address: str, flow_rate: str) -> str:
+    """Create a money flow using Superfluid.
+
+    Args:
+        wallet (Wallet): The wallet initiating the flow.
+        recipient (str): Recipient's wallet address.
+        token_address (str): Address of the token that will be streamed.
+        flow_rate (str): Rate of token flow in wei per second.
+        
+    Returns:
+        str: Confirmation of flow creation.
+    """
+    try:
+        invocation = wallet.invoke_contract(
+            contract_address="0xcfA132E353cB4E398080B9700609bb008eceB125",
```

Comment:
what networks are supported? Just `base-mainnet` and `base-sepolia` or more? Can we add details about network support to each of the action prompts.

Reply:
The CFA forwarder contract address is the same on all networks. I have added the relevant info to the prompt now as well.